src/main/java/net/atos/zac/policy/input/DocumentInput.java

```diff
@@ -14,7 +15,7 @@ public class DocumentInput extends UserInput {
     @JsonbProperty("document")
     private final DocumentData documentData;
 
-    public DocumentInput(final LoggedInUser loggedInUser, final DocumentData documentData) {
+    public DocumentInput(@NotNull LoggedInUser loggedInUser, final DocumentData documentData) {
```

Comment:
Do these @NotNull annotations really work here? I thought they only worked with a related @Valid annotation somewhere earlier in de callstack?

Reply:
Nope - totally not working. These are left from my first attempts. Will get rid of these.